common/network-common/src/main/java/org/apache/spark/network/server/StreamManager.java

```diff
@@ -42,9 +42,10 @@
    * The returned ManagedBuffer will be release()'d after being written to the network.
    *
    * @param streamId id of a stream that has been previously registered with the StreamManager.
+   * @param channel The connection used to serve chunk request.
```

Comment:
let's say more about this parameter, especially how it should be used. IIUC we need to track the channel states, and do some cleanup if the channel is inactive.